conda-recipe/meta.yaml

```diff
@@ -16,7 +16,6 @@ build:
 requirements:
   build:
     - python
-    - six
 
   run:
     - python
```

Comment:
In a google hangouts conversation with Tom, he says
` am inclined to keep them out of the conda deps right now because doct will be used by the server sides of mds/fs/...`

Fine by me. Merging.